.github/workflows/release.yml

```diff
@@ -57,6 +57,7 @@ jobs:
         uses: docker/setup-buildx-action@v1
         with:
           install: true
+          buildkitd-flags: --debug
```

Comment:
This and `BUILDKIT_PROGRESS` were added in attempt to get the Docker build to emit more logs during build, so we can see what vcpkg packages get installed.  I'm not sure if _both_ are needed, but didn't want to invest the build time in trying to game these.